libraries/chain/db_maint.cpp

```diff
@@ -809,7 +809,7 @@ void update_and_match_call_orders( database& db )
  *
  * @param db the database
```

Comment:
Please add doc for the new parameter.

By the way, I think it's better to name the new parameter `skip_check_call_orders` or something.